build.rs

```diff
@@ -5,10 +5,17 @@
 use std::env;
 use std::process::{Command, Stdio};
 
+
+#[cfg(target_os = "freebsd")]
+static MAKE_CMD: &'static str = "gmake";
+#[cfg(not(target_os = "freebsd"))]
+static MAKE_CMD: &'static str = "make";
+
+
 fn main() {
     let out_dir = env::var("OUT_DIR").unwrap();
     let target = env::var("TARGET").unwrap();
-    let result = Command::new("make")
+    let result = Command::new(MAKE_CMD)
```

Comment:
Maybe this should use the `$MAKE` env var if it's set (and fall back to the constant if it's not), so people on other BSDs or similar systems can more easily build without patching the build script.